### PR TITLE
nyx: unbreak for python 3.11, adopt.

### PR DESCRIPTION
**Pull request by steinex**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/nyx/template [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Frank Steinborn <steinex@nognu.de>"
 license="GPL-3.0-or-later"
 homepage="https://nyx.torproject.org/"
 distfiles="${PYPI_SITE}/n/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=88521488d1c9052e457b9e66498a4acfaaa3adf3adc5a199892632f129a5390b
+
+do_check() {
+	./run_tests.py
+}
```
Comment (paper42) [inline srcpkgs/nyx/template:18]: this doesn't fail when tests fail

Comment (paper42) [reply srcpkgs/nyx/template:18]: If you are using pytest, then you can try `--deselect` or patching them out.

Comment (steinex) [reply srcpkgs/nyx/template]: Sorry, I'm not able to bring this further than to disable tests altogether... I'm really no python test-ninja and this doesn't seem to use `pytest`.

**Comment by steinex** [comment]
Ready for review, then.

**Pull request merged**